src/components/Accordion/Accordion.test.tsx

```diff
@@ -446,4 +464,62 @@ describe('Accordion component', () => {
       expect(headings.length).toEqual(testItems.length)
     })
   })
+
+  describe('default headingLevel for AccordionItems', () => {
+    it('uses the default headingLevel and warns deprecation', () => {
+      const customTestItems: AccordionItemProps[] = [
+        {
+          title: 'First Amendment',
+          content: (
+            <p>
+              Congress shall make no law respecting an establishment of
+              religion, or prohibiting the free exercise thereof; or abridging
+              the freedom of speech, or of the press; or the right of the people
+              peaceably to assemble, and to petition the Government for a
+              redress of grievances.
+            </p>
```

Comment:
Maybe worth defining as a `const firstAmendmentContent` or something (same with the other accordion items), just to keep the test a little more brief. Not a huge deal since DAMP tests are good, but I think that'd be more than acceptable to factor out.

Reply:
smart